spec/requests/seller_gift_cards_request_spec.rb

```diff
@@ -44,6 +44,7 @@
 
       let(:expected_gift_card_latest_amount) { 30_00 }
       let(:expected_gift_card_original_amount) { 10_00 }
+      let(:expected_gift_card_last_updated) { Time.current + 1.day }
```

Comment:
nit: shouldn't this time be in the past? Not that it matters for a test 😂

Reply:
yeah that's fair. it's easier to do it this way though because the created_at/updated_at for the other gift cards are now(). so to make this row a current/past time, you'd have to update the created_at/updated_at for the rest of the other gift cards to be a time before that so that this row is the newest